docker-compose.override.yml

```diff
@@ -22,8 +22,9 @@ services:
       context: ./transcendence-app
       target: development
     volumes:
-      - ./transcendence-app:/transcendence-app
-      - /transcendence-app/node_modules
+      - ./transcendence-app/src:/transcendence-app/src:ro
+      - ./transcendence-app/test:/transcendence-app/test:ro
+      - ./transcendence-app/seeds:/transcendence-app/seeds:ro
```

Comment:
We mount only these subdirectories inside the container because if we mount the entire directory, Docker creates the `dist` and `node_modules` directories on the host during docker image build.

If we change any files outside these subdirectories, we rebuild the image using `make`.

Reply:
What do you think about letting docker create the detectors and then doing something like we do in `/scripts/generate-cmd.sh` lines 6 and 7 (passing MY_USER as an env var from the calling process)?

Reply:
Do you mean the `dist` and `node_modules` directories? With this change, Docker creates these directories only inside the image, and if we don't mount the entire transcendence-app directory, it doesn't create them on the host.